src/sage/combinat/integer_vectors_mod_permgroup.py

```diff
@@ -763,18 +788,192 @@ def __iter__(self):
             [3, 1, 1, 2]
             [3, 0, 2, 2]
             [2, 2, 2, 1]
+
+            Check that :issue:`36681` is fixed::
+            sage: I = IntegerVectorsModPermutationGroup(
```

Comment:
Indentation is wrong here: `Check ...` needs to go 4 spaces to the left.  Also, add a new blank line after `::`